### PR TITLE
Add cypress and a11y tests

### PR DESCRIPTION
**Pull request by matthis-d**

### CODE REVIEW

File: src/App.scss [hunk 1]
```diff
@@ -20,7 +20,7 @@ button {
   border-radius: 0.2rem;
   font-size: 1.2rem;
   text-decoration: none;
-  color: $base_white;
+  color: #ffffff;
```
Comment (Lokymil) [inline src/App.scss]: Why not modify $base_white color ?

File: src/_colors.scss [hunk 2]
```diff
@@ -9,10 +9,10 @@ $light_red: #f48fb1;
 $light_blue: #90caf9;
 $light_yellow: #ffe082;
 
-$base_grey: #607d8b;
+$base_grey: #546c78;
```
Comment (Lokymil) [inline src/_colors.scss:12]: Was there a contrast issue ?

File: src/components/interview/Theory.scss [hunk 3]
```diff
@@ -49,6 +49,7 @@
     &--link {
       text-align: center;
       margin-bottom: 6rem;
+      color: #ffffff;
```
Comment (Lokymil) [inline src/components/interview/Theory.scss]: $base_white (update it to #ffffff)

**Review by Lokymil — CHANGES_REQUESTED** [review]

File: src/App.scss (hunk 1 above)
Comment (matthis-d) [reply src/App.scss]: I just wanted to change color in button to have enough contrast in them, not to modify background or other elements.
What do you prefer @Lokymil?

File: src/_colors.scss (hunk 2 above)
Comment (matthis-d) [reply src/_colors.scss:12]: Yep

File: src/App.scss (hunk 1 above)
Comment (Lokymil) [reply src/App.scss]: Change $base_white so have fix contrast every where

File: src/components/interview/Theory.scss (hunk 3 above)
Comment (matthis-d) [reply src/components/interview/Theory.scss]: Done 😉

File: src/App.scss (hunk 1 above)
Comment (matthis-d) [reply src/App.scss]: Done

**Pull request merged**